static/js/unread_ops.js

```diff
@@ -10,13 +13,11 @@ import * as people from "./people";
 import * as recent_topics_ui from "./recent_topics_ui";
 import * as recent_topics_util from "./recent_topics_util";
 import * as reload from "./reload";
+import * as ui_report from "./ui_report";
 import * as unread from "./unread";
 import * as unread_ui from "./unread_ui";
 
 export function mark_all_as_read() {
-    unread.declare_bankruptcy();
-    unread_ui.update_unread_counts();
```

Comment:
Is the "mark as unread" modal still visible while this is running? I worry that it'll look like the operation isn't working.

Reply:
(I think this is the only call to `unread.declare_bankruptcy`? So if we're doing this, we should delete that code)

Reply:
Didn't delete `unread.declare_bankruptcy` since it was used extensively in tests, added a comment there.

> I worry that it'll look like the operation isn't working.

Added a banner for it, maybe it will help!

<img width="725" alt="image" src="https://user-images.githubusercontent.com/25124304/200574778-db9a78d5-680a-4293-8c68-3d0a7f93fd13.png">